llarp/messages/common.hpp

```diff
@@ -18,6 +18,24 @@ namespace
 
 namespace llarp
 {
+  namespace messages
+  {
+
+    inline std::string
+    serialize_response(oxenc::bt_dict supplement = {})
+    {
+      return oxenc::bt_serialize(supplement);
+    }
+
+    // ideally STATUS is the first key in a bt-dict, so use a single, early ascii char
+    inline auto STATUS_KEY = "!"s;
+    inline auto STATUS_OK = "OK"sv;
+    inline auto STATUS_TIMEOUT = "TIMEOUT"sv;
+    inline auto STATUS_ERROR = "ERROR"sv; // generic, unknown error (e.g. onion decrypt throw)
+    inline auto TIMEOUT_BT_DICT = serialize_response({{STATUS_KEY, STATUS_TIMEOUT}});
+    inline auto ERROR_BT_DICT = serialize_response({{STATUS_KEY, STATUS_ERROR}});
```

Comment:
I am indifferent to namespacing the statuses (since there are only a couple), but I can if you want.  STATUS_OK as a dict could be useful I guess; I was considering the case where there's more meat to the reply than that, but if a reply is a trivial "ok, done" rather than a response (e.g. an intro set) then I suppose having a pre-fab dict is ok.

Reply:
I'm sortof indifferent, was more floating some ideas since I was doing something similar with storage server. I'm fine with altering these types of things as we go, depending on what we use the most/what we don't use at all?

Reply:
> I'm fine with altering these types of things as we go, depending on what we use the most/what we don't use at all?

You're starting to sound like me.  And yeah, that's how I see it as well.